lib/controllers/usuario_controller.js

```diff
@@ -0,0 +1,29 @@
+import Usuario from '../models/usuario';
+import passport from 'passport';
+
+export const index = async (req, res) => {
+  const data = await Usuario.findAll({});
+  res.send({ data });
+};
+
+export const registro = (req, res, next) => {
+  passport.authenticate('registro', (err, data) => {
+    if (err) {
+      return res.status(400).json({ error: err.errors[0].message });
```

Comment:
Fue, lo mergeamos así, ¿pero qué pasa si hubo varios errores?

Tal vez podríamos devolverlos todos:
```suggestion
      return res.status(400).json({ error: err.errors.map(it => it.message) });
```